recipe/run_test.py

```diff
@@ -29,4 +28,10 @@
 else:
     test(package='time')
     test(package='wcs')
-    test(package='convolution')
\ No newline at end of file
+    test(package='convolution')
```

Comment:
They only test `time` `wcs` and `convolution` in the c-f feedstock, fyi

Reply:
I thought we typically tried to cover as many tests as possible within reason? If that's no longer the case, I'm happy to remove them.

Reply:
yep fair enough, I think that's good then. Since they're all passing